packages/components/src/components/date-picker/date-picker-calendar.tsx

```diff
@@ -1,13 +1,28 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { CSSTransition } from 'react-transition-group';
 import Calendar from '../calendar';
 import { useBlockScroll } from '../../hooks';
 
+type TDatePickerCalendar = {
+    value: string | null;
+    alignment?: string;
+    is_datepicker_visible: boolean;
+    calendar_el_ref: React.RefObject<HTMLElement>;
+    parent_ref: React.RefObject<HTMLElement>;
+    portal_id: string;
+    style: Record<string, string | number>;
+    placement: string;
+    onHover?: (Arg1: string) => void;
```

Comment:
@amir-deriv let's follow naming convention on our project: for functions we use camelCase, for the rest of the variables we use snake_case. Let's use a meaningful variable name `hovered_date` instead of `Arg1`. Please refer to https://github.com/binary-com/deriv-app/blob/master/docs/JavaScript/README.md#naming-conventions Please do the same for all the rest of places where you put `Arg` in this PR 🙂